content/04.results.md

```diff
@@ -41,3 +41,26 @@ We observed that including one label function of a mismatched type to distant su
 We found that usually adding random noise did not improve performance (Figure {@fig:auroc_random_label_function_performance} and Supplemental Figure {@fig:aupr_random_label_function_performance}).
 For the CbG edge type we did observe slightly increased performance via AUPR (Supplemental Figure {@fig:aupr_random_label_function_performance}).
 However, performance changes in general were smaller than those observed with mismatched label types.
+
+### Expanding Hetionet
+
+![
+Text-mined edges can recall a fair amount of existing edges as well as add a myraid amount of novel edges to Hetionet v1 
+This bar chart shows the number of edges we can successfully recall in green and shows the number of new edges that can be added in blue.  
+Percentages contained in parenthesis represent the fraction of edges we recalled over the total number of edges already contained in Hetionet v1.
+For example, within the compound treats disease edge we can recall 85% of existing edges as well as add 6,088 novel edges.
+](https://raw.githubusercontent.com/danich1/snorkeling/6e929e486537c5d105e393e20984b96910c96024/figures/edge_prediction_experiment/edges_added.png){#fig:hetionet_reconstruction}
+
+Sentences need to be translated into edge representations before they are incorporated into Hetionet v1.
```

Comment:
Not everyone will know what hetionet v1 is. This is much later in the paper than your intro. Give a sentence or two describing roughly how hetionet was built (databases, etc) and what the purposes of this evaluation were. See what I've done to the section title for a starting point.